torch/csrc/autograd/init.cpp

```diff
@@ -1254,11 +1254,24 @@ static PyObject* len_torch_dispatch_stack(PyObject* _unused, PyObject* args) {
   END_HANDLE_TH_ERRORS
 }
 
-PyObject* THPModule_increment_version(PyObject* _unused, PyObject* tensor) {
+PyObject* THPModule_increment_version(
+    PyObject* _unused,
+    PyObject* tensor_list) {
   HANDLE_TH_ERRORS
+  PyObject* iterator = PyObject_GetIter(tensor_list);
```

Comment:
Refcount for this and item below is missing

Reply:
thanks you - as per offline, I'm now wrapping both `PyObject_GetIter` and `PyIter_Next` outputs in a `THPObjectPtr`, so they are properly decref'd on destruction.